daemon/SecretsImpl/secretsrequestprocessor.cpp

```diff
@@ -1346,10 +1346,18 @@ Daemon::ApiImpl::RequestProcessor::userInput(
         // be considered to be "secure" in the secrets-storage sense.
 
         InteractionParameters::PromptText promptText = promptParams.promptText();
-        //: Inform the user that the application is requesting data. %1 is the application name, %2 is the prompt text supplied by the application.
-        //% "An application %1 is requesting input which will be returned to the application: %2"
-        promptText.setMessage(qtTrId("sailfish_secrets-user_input-la-data_request")
-                              .arg(callerApplicationId, promptText.message()));
+
+        if (applicationIsPlatformApplication) {
+            //: Inform the user that some system application is requesting data. %1 is the prompt text supplied by the application.
+            //% "System application is requesting input: %1"
```

Comment:
Perhaps "A system application is requesting input: %1" instead.

Reply:
done